Source/Phone/Call/VideoLicense.swift

```diff
@@ -22,67 +22,68 @@ import Foundation
 
 class VideoLicense {
     
-    static let sharedInstance = VideoLicense()
-    
+    private let callMetrics: CallMetrics
     private let userDefaults = UserDefaults.sharedInstance
     
-    func checkActivation(_ completion: @escaping (_ isActivated: Bool) -> Void) {
-        guard needActivation() else {
-            completion(true)
+    init(callMetrics: CallMetrics) {
+        self.callMetrics = callMetrics
+    }
+    
+    func checkActivation(completionHandler: ((_ isActivated: Bool) -> Void)?) {
```

Comment:
It became optional, and optional closures are always escaping. I was surprised it worked like that too!